modules/package-module.sh

```diff
@@ -86,7 +86,7 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
 	echo
 	echo "Installing ard-scene-select"
 	installrepo ard-scene-select module-prod https://github.com/GeoscienceAustralia/dea-ard-scene-select.git
-	#installrepo ard-scene-select  DSNS-109-fix          https://github.com/GeoscienceAustralia/dea-ard-scene-select.git
+	#installrepo ard-scene-select  dsg_dev          https://github.com/GeoscienceAustralia/dea-ard-scene-select.git
```

Comment:
i think it would be consistent with the other scripts such that we have a filter to check whether it is a production run or not.

Reply:
good idea